bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.tsx

```diff
@@ -60,12 +64,44 @@ interface VideoListItemProps {
   };
 }
 
+const renderPluginItems = (
+  streamId: string, userId: string,
+  pluginItems: UserCameraHelperInterface[],
+  bottom: boolean, right: boolean,
+) => {
+  if (pluginItems !== undefined) {
+    return (
+      <>
+        {
+          pluginItems.map((pluginItem) => {
```

Comment:
Missing the displayFunction filter here
```suggestion
          pluginItems.filter(
            (pluginItem) => (pluginItem.displayFunction?.({ userId, streamId }) ?? true),
          ).map((pluginItem) => {
```